aspnetcore/web-api/index/samples/3.x/Startup.cs

```diff
@@ -40,6 +40,7 @@ public void ConfigureServices(IServiceCollection services)
                     options.SuppressMapClientErrors = true;
                     options.ClientErrorMapping[StatusCodes.Status404NotFound].Link =
                         "https://httpstatuses.com/404";
+                    options.DisableImplicitFromServicesParameters = true;
```

Comment:
@Rick-Anderson , this property will only be available in .NET 7, is ok to use the 3.x sample?